test_grid.c

```diff
@@ -40,21 +40,31 @@ static void grid_cycle(G_GNUC_UNUSED GtkWidget *widget) {
 }
 
 static void grid_draw(GtkWidget *widget) {
+  cairo_t *cr;
   GdkWindow *win = gtk_widget_get_window(widget);
   gint w, h;
   gint i;
-  gint d;
```

Comment:
`d` is unused in this and some of the other tests. I couldn't find out why it was introduced, since it's in from the beginning of the Git history. But it seems safe for removal.